cfgov/unprocessed/css/molecules/translation-links.less

```diff
@@ -58,3 +63,36 @@ html[lang='ar'] ul.m-translation-links {
     }
   }
 }
+
+//fix issue with padding/margin and verticle rule appearing to the left of print icon
+html[lang='ar'] .m-social-media {
+  display: inline-block;
+  direction: rtl;
+
+  &_print {
+    padding-left: 0;
+    border-left: none;
+    padding-right: unit((@grid_gutter-width / 2) / @base-font-size-px, em);
+    border-right: 1px solid @black;
+  }
+
+  & .m-list .m-list_item:last-child {
+    margin-right: 0.6875em;
+  }
+
+  //Position share & print text above social media icons and remove right margin
+  @media screen and (max-width: 420px) {
+    & .m-list {
+      width: 100%;
+      padding-top: 0.6875em;
```

Comment:
We generally use the "unit" helper you see on line 75, so that it is easier to read what the pixel value is meant to be. 

 - Looks like this should be:

`padding-top: unit(11px / @base-font-size, em);`

 - As an aside, is 11px correct? We usually have increments of 5 pixels, so that looks a little suspect, but if it looks correct on the page that's fine.

 - You could consider using `rem` instead. I'm probably going to use `rem` going forward, but let me know if it's a bad idea.

Reply:
Chrome must have been giving me a measurement based off my screen res. I changed the padding to match other padding in the CSS ("unit(15px / @base-font-size-px, rem)"). The spacing looks fine to me.